src/main/java/org/opensearch/geospatial/index/mapper/xypoint/ParsedXYPoint.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.index.mapper.xypoint;
+
+import org.opensearch.geometry.Point;
+import org.opensearch.index.mapper.AbstractPointGeometryFieldMapper;
+
+/**
+ * represents a XYPoint that has been parsed by {@link AbstractPointGeometryFieldMapper.PointParser}
```

Comment:
nit: I think link should be for AbstractPointGeometryFieldMapper.ParsedPoint, same as actual class

Reply:
ack